### ISSUE TITLE
scrollToRow should scroll to an exact position

### ISSUE THREAD

**Issue by francescogior** [facebook/fixed-data-table#175]
It would be nice if the prop `scrollToRow` actually scroll the view to make it that row on top of the visibile rows.

Instead, i seems it is able to make the minimum effort to make that row visible, basically
* If the row is visible (in the middle of the view) it does nothing
* If the row index is greater than the greatest visible, it scrolls down and put the row at the bottom of the view
* If the row index is less than the smallest visible, it scrolls up and put the row on top of the view

![image](https://cloud.githubusercontent.com/assets/3280300/8256724/1d92c8fe-16a9-11e5-9f2a-e42a3c32f5f6.png)

**Comment by MandarinConLaBarba** [comment]
I agree. Seems like the most appropriate behavior would be to center the row vertically in the scrollable area.

**Comment by mitjacotic** [comment]
What is the a reason that `scrollToRow` [calls `scrollRowIntoView`](https://github.com/facebook/fixed-data-table/blob/master/src/FixedDataTableNew.react.js#L767) ?
Replacing that call with `scrollToRow()` would properly scroll the list so that desired row would be on the top of the list, but right now, like **francescogior** said, it only ensures that the row is somewhere in the visible viewport.
I think there should be option for both; current behaviour and also an option to scroll some row to the top of the list but currently latter doesn't seem to be doable without modifying the lib itself.
What about new prop, something like **scrollRowToTop** so devs could choose the one which suits them more?
